src/Sylius/Component/Core/Model/Product.php

```diff
@@ -355,4 +355,16 @@ public function getImage()
 
         return $this->getFirstVariant()->getImage();
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getImages()
+    {
+        if (null === $this->getFirstVariant()) {
+            return new ArrayCollection();
```

Comment:
@CarwynNelson Actually, it returns first ProductVariant from $variants collection, but product can have no variants at all, if not configured properly yet. Anyway, we will be adding images support on Product, so these methods will return images and first image from product itself. :)